src/Compiler/Checking/CheckExpressions.fs

```diff
@@ -4759,23 +4759,37 @@ and CrackStaticConstantArgs (cenv: cenv) env tpenv (staticParameters: Tainted<Pr
         error (Error(FSComp.SR.etTooManyStaticParameters(staticParameters.Length, unnamedArgs.Length, namedArgs.Length), m))
 
     let argsInStaticParameterOrderIncludingDefaults =
+        let adjustRangeForQuotes (arg: SynType) =
+            match arg with
+            | SynType.StaticConstant (SynConst.String (_, kind, m), _) ->
+                let startOffset, endOffset =
+                    match kind with
+                    | SynStringKind.Regular -> 1, 1
+                    | SynStringKind.Verbatim -> 2, 1
+                    | SynStringKind.TripleQuote -> 3, 3
```

Comment:
Triple quotes verbatim strings do not have their own kind, so the range becomes off by one. Not sure it's worth adding.

Reply:
@abonie  : I assume you plan to touch the parsing and tree representation of string kinds anyway, could you account for this as well please?
i.e. the ability to cover the range(s) involved precisely, and perharps even Trivia(s) to give ranges per line in case of multiline strings.